src/DynamicFormControl.vue

```diff
@@ -1,93 +1,89 @@
 <template>
     <b-col :md="colWidth">
-        <label v-if="formControl.label">{{formControl.label}}
+        <label v-if="formControl">{{formControl.label}}
             <span v-if="formControl.helpText"
                   class="icon-small"
                   v-tooltip="formControl.helpText">
-                <help-circle-icon></help-circle-icon>
+               <vue-feather type="help-circle"></vue-feather>
             </span>
             <span v-if="formControl.required && !readonly" class="small" :class="{'text-danger': valueIsEmpty(formControl.value)}">({{requiredText}})</span>
```

Comment:
I don't think this line should be here, because currently it is displaying (required) over the form field. I think once next to the form label is sufficient

Reply:
okay yh no worries, then maybe a future ticket?